Dockerfile

```diff
@@ -220,11 +220,15 @@ RUN apt-get update && \
 
 USER tenzir:tenzir
 COPY demo-node /demo-node
-env TENZIR_AUTOMATIC_REBUILD=0
-env TENZIR_ALLOW_UNSAFE_PIPELINES=true
+ENV TENZIR_AUTOMATIC_REBUILD=0 \
+    TENZIR_ALLOW_UNSAFE_PIPELINES=true \
+    TENZIR_DB_DIRECTORY="/var/lib/tenzir-demo"
+COPY --from=tenzir-ce --chown=tenzir:tenzir /var/lib/tenzir/ /var/lib/tenzir
+COPY --from=tenzir-ce --chown=tenzir:tenzir /var/lib/tenzir/ /var/lib/tenzir-demo
 RUN /demo-node/setup.bash
+ENV TENZIR_DB_DIRECTORY="/var/lib/tenzir"
 
-ENTRYPOINT ["tenzir-node"]
+ENTRYPOINT ["sh", "-c", "2>&1 echo Copying demo data... && cp -rf /var/lib/tenzir-demo/. /var/lib/tenzir/ && tenzir-node"]
```

Comment:
why reset demo nodes on restart?

Reply:
ah, I guess it's already what it does. Did you have a use-case where you want to mount a persistent volume? If not I would rather leave it as is instead of adding this additional complexity and weight.

Reply:
I tried to use the demo node image in our Docker Compose setup, and that didn't work without this change because it mounts by default. This makes it work transparently.

Reply:
which docker compose are you referring to exactly?

Reply:
The one at the root of this repository.

Reply:
If you really have a use for it, lets leave it. But if we can spare it I would really not add this extra complexity.

Reply:
If the main goal is to test the demo node image, I really think we should have a dedicated setup for that instead of adding a workaround that impacts all prod runs 😄

Reply:
Last remark: if we keep this copying logic, I would just move it back to an entrypoint script for readability and extensibility. Calling `exec` is also always nicer to have the `tenzir-node` process be the root process.

Reply:
I can move it back out, I don't mind. Would you prefer that?

Reply:
Then you can also keep the custom entrypoint.

Reply:
Changed in 04c662bccae42c912266603fa13dcc5bf065e339